pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[tool.black]
+skip-magic-trailing-comma = true
```

Comment:
Why this option?

Reply:
Good question,
It's one of the few times where black look at the previous formatting and uses this information. Furthermore, people who haven't read the black documentation trigger the magic comma without knowing about it. We can always remove this option if you think the magic comma is useful, I don't really mind:
https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma